pilot/pkg/bootstrap/server.go

```diff
@@ -194,6 +214,7 @@ type PilotArgs struct {
 	KeepaliveOptions         *istiokeepalive.Options
 	// ForceStop is set as true when used for testing to make the server stop quickly
 	ForceStop bool
+	Chiron    ChironArgs
```

Comment:
See above - this should go to api mesh config, and use user-friendly name.

Reply:
Done.